examples/web/src/components/VideoPlayer.js

```diff
@@ -0,0 +1,180 @@
+export default {
+    name: 'VideoPlayer',
+    components: {},
+    data() {
+        return {
+
+            newpath: "http://localhost:8080",
```

Comment:
What is this used for?

Reply:
Removed. It was used in the <video> tag to indicate the source when using tcpserversink. I suspect.